test_security/CMakeLists.txt

```diff
@@ -0,0 +1,254 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(test_security)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+option(SECURITY "Activate security" OFF)
```

Comment:
Just to consider for naming this options: a lot of options like this start with `ENABLE_`.

Reply:
This option is the one use by [fastrtps](https://github.com/eProsima/Fast-RTPS/blob/5c910f7643c26f7d4764e4d675f23abeadd3c2ee/CMakeLists.txt#L208) and in our CI so if we change it here we would need to check if fast-rtps has been built with security and pass an additional argument to our ci jobs